.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -1,36 +1,35 @@
 name: Documentation request
 description: Raise an issue related to content.
-title: "📝 Doc:"
-labels: "documentation, to sort"
+title: "📝 Doc: "
+labels: ["documentation", "to sort"]
 body:
   - type: markdown
     attributes:
       value: |
-        * Please check that there is not an existing open [issue](https://github.com/tracim/tracim/labels/documentation) before you create a new one.
+        Thanks for wanting to improve Tracim!
+        Before you submit, please make sure the documentation change hasn't already been suggested by searching [open](https://github.com/tracim/tracim/issues) and [closed](https://github.com/tracim/tracim/issues?q=is%3Aissue+is%3Aclosed).
   - type: input
+    id: contents
     attributes:
-      label: What content on github.com/tracim/tracim is affected?
-      placeholder: "github.com/tracim/tracim"
+      label: What content on Tracim is affected?
```

Comment:
Not sure if this is clear.
The documentation issue template is related to the documentation on this repository, not the one on Tracim.

Reply:
Can't it be to both?
I mean, the documentation is accessible to the repository but is initially on Tracim, the fact that the user have to put an URL address is just disturbing me

Reply:
I don't know, if it seems fine to you it's ok for me :)